services/Facebook.filters.js

```diff
@@ -1,4 +1,4 @@
-export function removeHelpButtons(document) {
+export async function removeHelpButtons(document, url) {
```

Comment:
Note: I had to convert filters into `async` functions to be able to execute fetch etc into them.